scalafmt-dynamic/src/test/scala/tests/ScalafmtVersionSuite.scala

```diff
@@ -51,6 +55,11 @@ class ScalafmtVersionSuite extends FunSuite {
       ScalafmtVersion
         .parse("2.1.0-RC1-M4") == Left(InvalidVersionException("2.1.0-RC1-M4"))
     )
+    assert(
+      ScalafmtVersion.parse("2.0.0-RC1-SNAPSHOT") == Left(
+        InvalidVersionException("2.0.0-RC1-SNAPSHOT")
```

Comment:
Why is it invalid?

Reply:
Oh, sorry, it's my mistake, it should be valid.